### PR TITLE
Take progress pictures

### PR DESCRIPTION
**Pull request by statham**
addresses #23 

fitnesstaskswalkthrough

EDIT:
added firebase cloud storage for the images taken through the app. I did this so that when we attempt to display the pictures in the summary tab, we can just pull them from firebase instead of trying to do some magic with the users gallery photos

### CODE REVIEW

File: app/src/main/java/com/kyphipraniti/fitnesstasks/fragments/CalendarFragment.java [hunk 1]
```diff
@@ -105,6 +179,52 @@ private void launchAddTaskFragment() {
...
     }
 
+    private void launchTakePhotoFragment() {
+        Intent intent = new Intent(MediaStore.ACTION_IMAGE_CAPTURE);
+        // Create a File reference to access to future access
+        photoFile = getPhotoFileUri(photoFileName);
+
+        // wrap File object into a content provider
+        // required for API >= 24
+        // See https://guides.codepath.com/android/Sharing-Content-with-Intents#sharing-files-with-api-24-or-higher
```
Comment (pranitishettiwar) [inline app/src/main/java/com/kyphipraniti/fitnesstasks/fragments/CalendarFragment.java]: Do we need these comments ?

**Comment by pranitishettiwar** [comment]
ohh I was like why are there 2 circles and black background, I guess it's due to the emulator. Have you tried it on real device ?

**Comment by Vsin** [comment]
This looks good to me!

**Review by Vsin — APPROVED** [review]
LGTM

**Comment by statham** [comment]
@pranitishettiwar updated code with your changes

@Vsin you should take another look since i just added another service to our app 😬

**Comment by pranitishettiwar** [comment]
Great!! 🚀

**Review by pranitishettiwar — APPROVED** [review]

**Pull request merged**